probability.ipynb

```diff
@@ -3015,10 +3014,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We observe that variable elimination was faster than enumeration as we had expected but the gain in speed is not a lot, in fact it is just about 30% faster.\n",
+    "In this test case we observe that variable elimination it's slower than what we expected. It has something to do with number of threads, how python tries to optimize things and  this happens because the network is very small, with just 5 nodes. The `elimination_ask` has some critical point and some optimizations must be perfomed as seen above.\n",
```

Comment:
It should be 'is slower' instead of 'it's slower'. Also, it's best if you capitalize 'Python'.